aws-integrations/ecs-fargate/ecs-fargate-cf.yaml

```diff
@@ -226,9 +226,11 @@ Resources:
             - ContainerPort: 4317
               HostPort: 4317
               Protocol: tcp
+              AppProtocol: grpc
             - ContainerPort: 4318
               HostPort: 4318
               Protocol: tcp
+              AppProtocol: grpc
```

Comment:
After have a quick look at the OTEL specs, I'm thinking it should actually be `grpc` for port 4317 as we have done here, but `http` for port 3418? I just committed back the changes as we did them to make it work, but let me know if this should be changed if that is the case.

Reply:
Yes, http would be correct for port 4318.